pkg/alignchecker/alignchecker.go

```diff
@@ -143,3 +145,86 @@ func check(name string, toCheck []reflect.Type, structs map[string]structInfo, c
 
 	return nil
 }
+
+// Read ELF symbols from .rodata and compare the values against given toCheck values
+func valueCheck(pathToObj string, toCheck map[string]reflect.Value) error {
+	f, err := elf.Open(pathToObj)
+	if err != nil {
+		return fmt.Errorf("ELF failed to open %s: %s", pathToObj, err)
+	}
+	defer f.Close()
+
+	elf_syms, err := f.Symbols()
```

Comment:
Nit: In Golang, we generally [follow camel case for variable names](https://golang.org/doc/effective_go.html#mixed-caps). Plus, this will help keep the casing consistent in this function. :)

Reply:
Thanks!

It seems that `golint` captures a lot of these, so hopefully I'll avoid them in the future.